docs/release/cut-sw-release.md

```diff
@@ -310,17 +310,44 @@ NON_UPCOMING_VERSIONS="<NON-UPCOMING VERSION(S)>"
 ```
 ```bash
 for ver in $NON_UPCOMING_VERSIONS; do
-    major_ver=`sed 's/.[0-9]*$//' <<< $ver`
+    major_ver="${ver%.*}"
     mv /tmp/$ver /usr/local/repo/tarball-install/$major_ver/
     rm -f /usr/local/repo/tarball-install/$major_ver/*latest*
 done
 /root/mk-sims.sh
 for ver in $NON_UPCOMING_VERSIONS; do
-    major_ver=`sed 's/.[0-9]*$//' <<< $ver`
+    major_ver="${ver%.*}"
     ls -l /usr/local/repo/tarball-install/$major_ver/*latest* # verify the symlinks are correct
 done
 ```
 
+Upload the tarballs with the following procedure:
+
+!!! note
+    If you do not have access to the tarball repository at UNL (osgcollab@hcc-osg-software.unl.edu),
+    please forward your public SSH key to Derek Weitzel.
```

Comment:
This may not be decided yet but what if Derek is OOO? Is there a list that we can use instead?

Also, let's move this note to the top of the section so users aren't doing a partial client tarball update and copy its text to the `Requirements` section up top.